examples/solids/elasticity.c

```diff
@@ -47,7 +49,11 @@ int main(int argc, char **argv) {
   MPI_Comm       comm;
   // Context structs
   AppCtx         app_ctx;                  // Contains problem options
-  Physics        phys;                     // Contains physical constants
+  Physics        phys =
+    NULL;              // Contains physical constants - Neo Hookean
+  Physics_MR     phys_MR =
+    NULL;           // Contains physical constants - Mooney-Rivlin
```

Comment:
Strange style. I'm not sure I see why this should be on two lines each.

Reply:
I think it was astyle handling the long comment, but I want to reactor this anyway (ASAP, but in a new PR).